src/Forms/Main.cs

```diff
@@ -19081,19 +19097,50 @@ private void exportCustomTextFormatToolStripMenuItem_Click(object sender, EventA
 
         private void PasteIntoActiveTextBox(string s)
         {
-            if (tabControlSubtitle.SelectedIndex == TabControlSourceView)
+            if (SubtitleListview1.SelectedItems.Count > 1)

```

Comment:
Sorry, but if you select 2 lines in the list view and go to the source view and paste... 
Also, saving+restoring the textBox.SelectionStart in your new helper method would be nice :)